src/plugins/cuda_gds/gds_utils.cpp

```diff
@@ -51,7 +51,7 @@ nixl_status_t gdsUtil::registerBufHandle(void *ptr,
 
     status = cuFileBufRegister(ptr, size, flags);
     if (status.err != CU_FILE_SUCCESS) {
-        std::cerr << "Warning: Buffer registration failed - will use compat mode\n";
+        NIXL_WARN << "Warning: Buffer registration failed - will use compat mode";
```

Comment:
maybe
```suggestion
        NIXL_WARN << "Buffer registration failed - will use compat mode";
```
as afaiu `warning` will be printed by absl